feat: optionally disable Vercel's insights and analytics (#1362)
bigcommerce/catalyst: .changeset/flat-tables-hang.md

---
"@bigcommerce/catalyst-core": minor
---
If app is not running on Vercel's infra, `<Analytics />` and `<SpeedInsights />` are not rendered.

> Opt-in to disable vercel analytics and speed insights by setting the following env vars to `true`

Reviewer: ```suggestion
Opt-out of vercel analytics and speed insights by setting the following env vars to `true`
```
